src/modular/monty_form.rs

```diff
@@ -37,33 +37,38 @@ impl<const LIMBS: usize> MontyParams<LIMBS> {
     ///
     /// Returns `None` if the provided modulus is not odd.
     pub fn new(modulus: &Uint<LIMBS>) -> CtOption<Self> {
```

Comment:
Should we just take `Odd` here thus removing the need to return `CtOption`?

Reply:
Yeah, seems like a good idea

Reply:
Impl'd in 7e511d7 but it looks like it needs a rebase